src/App.js

```diff
@@ -1,79 +1,74 @@
 import  React, { useEffect, useState } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-       function App() {
-               
+       function App() {          
   const [todoList, setTodoList]= useState (
- 
   JSON.parse(localStorage.getItem('savedTodoList')) || []
-
-  )};
-
+  );
   const [isLoading, setIsLoading] = useState(true);
-  
-    async function fetchData (
-   
-        
-        useEffect = () => {
+
+        useEffect = (() => {
          fetchData () 
         }, []);
     
-   
-            const options = {
+    async function fetchData () {
+      try {          
+      const options = {
               method: 'GET',
               headers: {
                 Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
-            }},
+            },
+          };
           
             
-     url = (`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}\`,{
-          
-            });
-             try {    
-          const response = await fetch (url, options);
-               console.log(response);
-               const data = await response.json();
-               console.log("API Data:", data);
-             };
-
+          const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+             
+           const response = await fetch (url, options);
+              
              if (!response.ok) {
               const message = ('Error occurred: ${response.status}');
               throw new Error(message);               
-            };
+            }
       
-           
+            const data = await response.json();
+            console.log("Data:", data); 
             
-            const todos = data.records.map((Todo) => {
+            const todos = data.records.map((records) => ({
               Id: records.Id,
               Title: records.fields.Title
               
-           }
-             return (todos)      
-            
+           }));
+               
+             console.log("Todos:", todos);
```

Comment:
Take a look at how you receive the data object
<img width="660" alt="Screen Shot 2023-06-12 at 10 25 42 AM" src="https://github.com/Tugeeta/react-todo/assets/11164899/08ff8492-7bc4-4033-bf9e-28df925329da">

You don't need to capitalize the id and title properties of the todos obj. Use them exactly as you receive them in the data obj.